plugins/woocommerce/client/blocks/tests/e2e/tests/accordion/accordion.block_theme.spec.ts

```diff
@@ -36,13 +36,16 @@ test.describe( `${ blockData.slug } Block`, () => {
 		} ) => {
 			await requestUtils.setFeatureFlag( 'experimental-blocks', false );
 			await admin.createNewPost();
+
+			try {
+				await editor.insertBlock( { name: blockData.slug } );
+			} catch ( _error ) {
+				// noop
+			}
+
 			await expect(
-				editor.insertBlock( { name: blockData.slug } )
-			).rejects.toThrow(
-				new RegExp(
-					`Block type '${ blockData.slug }' is not registered.`
-				)
-			);
+				await editor.getBlockByName( blockData.slug )
+			).toBeHidden();
```

Comment:
I'm just curious - in WordPress 6.8 this exception isn't raised anymore?

Reply:
It's different - Instead of a JS error in v6.7, it now throws a handled error in the canvas stating that the block is not supported. Checking if the block simply isn't there works for both scenarios.